controllers/v2/fcm_tokens/api.py

```diff
@@ -0,0 +1,81 @@
+import logging
+from repository.fcm_token_repository import FCMTokenRepository
+from flask_restful import Resource, reqparse
+from controllers.v2.v2_blueprint import v2_api
+
+parser = reqparse.RequestParser()
+parser.add_argument('userId', type=int, required=True, help ="userId must be provided.")
+parser.add_argument('token', type=str, required=True, help ="Token must be provided.")
+parser.add_argument('device_type', type=str, required=True, help ="DeviceType must be provided.")
+
+{
+  "userId": "<USER_ID>",
+  "token": "<FCM_TOKEN>",
+  "device_type": "<DEVICE_TYPE>"
+}
+
+
+class FCMToken(Resource):
+
+    token_repository: FCMTokenRepository
+
+    def __init__(self, token_repository: FCMTokenRepository = FCMTokenRepository()):
+        self.token_repository = token_repository
+
+    def post(self):
```

Comment:
These two endpoints must be authenticated, and they need to ONLY be accessible to the user who is authenticated. Unlike the other endpoints, do _not_ allow the root admin to access this resource for anyone but themselves.

As it currently stands, anyone could send a token to the server impersonating another user. You should be making assessments as to who should reasonably be able to access a resource as you are designing it.

Reply:
Please also define the output format using the `@marshal_with` annotation and model used elsewhere

Reply:
I'm not very sure, how to implement this properly since the method returns a message response.